superset-frontend/src/components/AsyncAceEditor/index.tsx

```diff
@@ -150,7 +151,8 @@ export default function AsyncAceEditor(
             mode={mode}
             theme={theme}
             tabSize={tabSize}
-            {...props}
+            defaultValue={defaultValue}
+            {...{ ...props, value: props.value || '' }}
```

Comment:
The syntax here looks alien. 😄 

Why not:

```suggestion
            {...props}
            value={props.value || '' }
```

Reply:
ha, yeah, that was my first implementation, but often people like to see the "rest" props at the end, and so I anticipated someone changing this order in the context of "cleanup". I think I found a way around this.. lmk what you think.